dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/statistics/StatisticsTransformer.java

```diff
@@ -410,5 +411,18 @@ private void addDisplayListing(Division mainDiv, StatisticsListing display)
 		}
 
 	}
+    
+    /**
+    * Gets the number of results to display from usage-statistics.cfg
+    * Returns 10 if there is no such entry in usage-statistics.cfg
+    * @return number of results to display
+    */
+    private int chooseResultCount() {
+        Integer numItems = ConfigurationManager.getIntProperty("usage-statistics", "itemcount.usage");
```

Comment:
Should use `ConfigurationService` instead of `ConfigurationManager` to support the new DSpace 6 Configuration System: https://wiki.duraspace.org/display/DSDOC6x/Configuration+Reference